plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -357,6 +356,53 @@ func TestGetRoundTripper(t *testing.T) {
 	}
 }
 
+func TestGetRoundTripperToken(t *testing.T) {
+	const wantBearer = "token from file"
+
+	file, err := os.CreateTemp("", "token_")
+	require.NoError(t, err)
+	defer func() { assert.NoError(t, os.Remove(file.Name())) }()
+
+	_, err = file.Write([]byte(wantBearer))
+	require.NoError(t, err)
+	require.NoError(t, file.Close())
+
+	rt, err := getHTTPRoundTripper(&config.Configuration{
+		ConnectTimeout: time.Second,
+		TokenFilePath:  file.Name(),
+	}, nil)
```

Comment:
@ChillOrb I made a few tweaks
* simplify temp file creation
* make sure to call file.Close() after writing content to flush the write
* change ConnectTimeout to 1sec

Reply:
and use const for wantBearer